app/views/ips/_table.html.erb

```diff
@@ -23,7 +23,7 @@
   <tbody class="govuk-table__body" id="ips-table">
     <% location.ips.each do |ip| %>
       <tr class="govuk-table__row">
-        <td class="govuk-table__cell govuk-!-width-one-third"><%= ip.address %></td>
+        <td class="govuk-table__cell govuk-!-width-one-third" id="ip-address"><%= ip.address %></td>
```

Comment:
Is this going to print #ip-address to the page multiple times? A class instead perhaps?

Reply:
Its just giving the `td` an `id` attribute, so nothing changes in the view. I went for `id` selector over `class` as I consider classes for css and ids for javascript and testing.

Reply:
Think it's against the rules to have more than one id with the same name on a page no?